### PR TITLE
Integration test works with existing GCP PostgresSQL

### PR DESCRIPTION
**Pull request by chaodaiG**
- Allow working with existing DB
- Add script bridging terraform with integration test

This PR has been tested with the database provisioned by terraform. Follow up would be figuring out how to test servers provisioned by terraform

Part of: #723

### CODE REVIEW

**Comment by whaught** [comment]
@assign @sethvargo

File: internal/integration/helpers.go [hunk 1]
```diff
@@ -130,6 +133,22 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 	}
 
 	db := database.NewTestDatabase(tb)
+	if v := os.Getenv("DB_NAME"); v != "" && !testing.Short() {
```
Comment (sethvargo) [inline internal/integration/helpers.go:136]: Why is this needed? I think the database util already sets up the secret manager and stuff, no?

Comment (chaodaiG) [reply internal/integration/helpers.go:136]: https://github.com/google/exposure-notifications-server/blob/2dfbe2f7cee56b5f0d6b5b66911d03f2ab29de1c/internal/database/database_util.go#L48
I don't think database_util.go sets up secret manager, not database from environment either
